src/ListActions/ListActions.js

```diff
@@ -263,166 +137,93 @@ var ListActionsDrawer = kind(
-		* vertically. In this case, an outer scroller is enabled; the outer scroller scrolls
-		* all groups vertically, and the `FittableRows` are reset to natural size based on
-		* their content, effectively disabling any scrollers contained within, to prevent
-		* nested scrolling.
-		*
-		* Note that the vertical stacking capability poses a limitation on using
-		* `moon/DataList`. Since `moon/DataList` must always be allowed to scroll, it is
-		* not suitable for use in a stacked scenario in which only one outer scroller is
-		* used. As such, it cannot be used within a `ListActions` that may need to stack
-		* vertically.
-		*
-		* Each group should have a string value set for the `action` property, as this will
-		* be passed in all events that bubble from the `ListActions`, to allow the user to
-		* identify which category changed.
-		*
-		* @type {Object[]}
-		* @default null
-		* @public
-		*/
-		listActions: null,
-
-		/**
-		* Source URL for icon image.
-		*
-		* @type {String|module:enyo/resolution#selectSrc~src}
-		* @default ''
-		* @public
-		*/
-		iconSrc: '',
-
-		/**
-		* Icon name to be used by the activator button (as in {@link module:moonstone/Icon~Icon} and
-		* {@link module:moonstone/IconButton~IconButton}).
-		*
-		* @type {String}
-		* @default ''
-		* @public
-		*/
-		icon: '',
-
-		/**
-		* By default, list action menus are 300px wide. Set this to `true` to instead have
-		* the menus be proportionally sized within the available space. Note that a minimum
-		* width of 300px is still respected; if all menus don't fit horizontally, they will
-		* be stacked vertically.
-		*
-		* @type {Boolean}
-		* @default false
-		* @public
-		*/
-		proportionalWidth: false,
-
-		/**
-		* The background-color opacity of the {@link module:moonstone/ListActions~ListActions}' activator
-		* (which is a {@link module:moonstone/IconButton~IconButton}). Please see the valid values defined by
-		* {@link module:moonstone/Button~Button#backgroundOpacity}.
-		*
-		* @type {String}
-		* @default 'opaque'
-		* @public
-		*/
-		backgroundOpacity: 'opaque'
-	},
+	iconSrc: '',
 
 	/**
-	* @private
+	* The background-color opacity of the {@link module:moonstone/ListActions~ListActions}' activator
+	* (which is a {@link module:moonstone/IconButton~IconButton}). Please see the valid values defined by
+	* {@link module:moonstone/Button~Button#backgroundOpacity}.
+	*
+	* @type {String}
+	* @default 'opaque'
+	* @public
 	*/
-	events: {
-		onShow: '',
-		onShown: '',
-		onHide: '',
-		onHidden: '',
-		onRequestCreateListActions: '',
-		onListActionOpenChanged: ''
-	},
+	backgroundOpacity: 'opaque',
 
 	/**
-	* @private
+	* CSS classes to apply to adjust width of each actions.
+	*
+	* @type {String}
+	* @default
+	* @public moon-list-actions-popup-width
 	*/
-	components:[
-		{name:'activator', kind: IconButton, classes: 'moon-list-actions-activator', ontap: 'expandContract'}
-	],
+	actionWidthClasses: 'moon-list-actions-popup-width',
```

Comment:
Should this really be a public, configurable property? I don't think we'd really want anyone setting a different value for this since we've included it in our CSS. If the width is configurable, should it be set by a LESS variable instead?